src/category_theory/instances/Top/open_nhds.lean

```diff
@@ -0,0 +1,43 @@
+import category_theory.instances.Top.opens
+
+open category_theory
+open category_theory.instances
+open topological_space
+
+universe u
+
+namespace topological_space.open_nhds
+variables {X Y : Top.{u}} (f : X ⟶ Y)
+
+def open_nhds (x : X.α) := { U : opens X // x ∈ U }
+instance open_nhds_category (x : X.α) : category.{u+1} (open_nhds x) := begin unfold open_nhds, apply_instance end
+
+def inclusion (x : X.α) : open_nhds x ⥤ opens X :=
+{ obj := λ U, U.val,
+  map := λ U V i, i }
+
+@[simp] lemma inclusion_obj (x : X.α) (U) (p) : (inclusion x).obj ⟨U,p⟩ = U := rfl
+
+def map (x : X) : open_nhds (f x) ⥤ open_nhds x :=
+{ obj := λ U, ⟨(opens.map f).obj U.1, by tidy⟩,
```

Comment:
Hmm, I'm hesitant to make a coercion for such a small reason. Maybe later?

Reply:
Sure, maybe later. In general I want to experiment with making forgetful functors into coercions. I hope it will make some parts read a lot more natural...